lib/node_modules/@stdlib/constants/float32/min-base10-exponent-subnormal/include/stdlib/constants/float64/min_base10_exponent_subnormal.h

```diff
@@ -0,0 +1,27 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef STDLIB_CONSTANTS_FLOAT32_MIN_BASE10_EXPONENT_SUBNORMAL_H
+#define STDLIB_CONSTANTS_FLOAT32_MIN_BASE10_EXPONENT_SUBNORMAL_H
+
+/**
+* Macro for the minimum base 10 exponent for a subnormal single-precision floating-point number.
+*/
+#define STDLIB_CONSTANT_FLOAT64_MIN_BASE10_EXPONENT_SUBNORMAL -45
```

Comment:
Should be `FLOAT32`. We also need to move the header file, as it currently resides in a `float64` directory.

```suggestion
#define STDLIB_CONSTANT_FLOAT32_MIN_BASE10_EXPONENT_SUBNORMAL -45
```

Reply:
pushed the changes! thanks @Planeshifter